pages/login.vue

```diff
@@ -1,13 +1,5 @@
 <template>
     <div class="h-full">
-        <TopNav />
-        <div class="flex flex-col md:flex-row h-full">
-            Login!
-        </div>
+        <LoginForm />
```

Comment:
Just curious, what's the reasoning for moving everything into a component instead of just using the page?

Reply:
I was thinking we might reuse the component in the auth0 login....but now I'm not sure honestly if it's going to be in more than one place.
 I personally try to keep the pages minimal as a habit, but that's just a personal preference.